lib/matplotlib/backends/backend_wx.py

```diff
@@ -1697,9 +1741,107 @@ def set_function(self, string):
 
 
 # tools for matplotlib.backend_managers.ToolManager:
-# for now only SaveFigure, SetCursor and Rubberband are implemented
-# once a ToolbarWx is implemented, also FigureManagerWx needs to be
-# modified, similar to pull request #9934
+
+class ToolbarWx(ToolContainerBase, wx.ToolBar):
+    def __init__(self, toolmanager, parent, style=wx.TB_HORIZONTAL):
+        ToolContainerBase.__init__(self, toolmanager)
+        wx.ToolBar.__init__(self, parent, -1, style=style)
+        self._toolitems = {}
+        self._groups = {}
+        self._last = None
+
+    def add_toolitem(
+        self, name, group, position, image_file, description, toggle):
+
+        before, group = self._add_to_group(group, name, position)
+        idx = self.GetToolPos(before.Id)
```

Comment:
where is this method defined? Pep8 lowercase

Reply:
I guess it comes from wx.Toolbar, forget the previous comment

Reply:
Sorry, it went out as review... I didn't know how to cancel the "start a review"